libs/styles/breakpoint-theme.css

```diff
@@ -0,0 +1,193 @@
+
+  
+/* Static Links */
+.con-block.static-links[class*="dark-"] a:not(.con-button),
+.static-links .con-block[class*="dark-"] a:not(.con-button),
+.con-block.static-links[class*="light-"] a:not(.con-button),
+.static-links .con-block[class*="light-"] a:not(.con-button),
+.con-block.static-links[class*="dark-"] a:not(.con-button):hover,
+.static-links .con-block[class*="dark-"] a:not(.con-button):hover,
+.con-block.static-links[class*="light-"] a:not(.con-button):hover,
+.static-links .con-block[class*="light-"] a:not(.con-button):hover {
+  color: inherit;
+}
+
+/* mobile ONLY */
+@media (max-width: 600px) {
```

Comment:
Since tablet applies starting 600, shouldn't this be `599px`?

Reply:
Looks like this is set on a number of files as is, so i'm leaving this alone for now.